packages/expo-localization/plugin/src/utils.ts

```diff
@@ -0,0 +1,101 @@
+export function appendContentsInsideDeclarationBlock(
```

Comment:
I'm not too confident shipping code like this without some unit tests – any chance you could add some simple test cases to make sure this behaves correctly (and also it will document it a bit better)?

Reply:
I agree it would be best to reexport it from the expo package – we can perhaps land that before we land this change?

I can take a look at that.

Reply:
https://github.com/expo/expo/pull/37885 – after this lands you should be able to do
`AndroidConfig.CodeMod.appendContentsInsideDeclarationBlock`